pmultiqc/modules/quantms/sparklines.py

```
@@ -250,7 +272,9 @@ def make_table(dt, maxValue):
                         if cmatches[cfck]:
                             badge_col = cfc[cfck]
                 if badge_col is not None:
-                    valstring = '<span class="badge" style="background-color:{}">{}</span>'.format(badge_col, valstr)
+                    valstring = '<span class="badge" style="background-color:{}">{}</span>'.format(
```

Ruff (0.8.2)

Local variable `valstring` is assigned to but never used

Remove assignment to unused variable `valstring`

(F841)